fedmsg.d/fas_credentials.py

```diff
@@ -1,8 +1,6 @@
-import getpass
```

Comment:
The first thing I did when I got started was to copy the config to ~/.fedmsg.d and write a user unit file to run it under systemd, and I hit this. Maybe I shouldn't have, but I did.

I can put it back and either hack around it in the unit tests (which I don't think will be easy) or make a note that you can't run the unit tests from the project root, but I don't like that solution. To me, it's incredibly weird to have code that blocks for user input being executed as part of an ``import`` statement. Every set of tests is also going to have to account for this, so we're _still_ going to have to document something about this.